app/Resources/FOSUserBundle/serializer/Model.User.yml

```diff
@@ -0,0 +1,11 @@
+FOS\UserBundle\Model\User:
```

Comment:
Not really IMO, check docs: http://jmsyst.com/bundles/JMSSerializerBundle/master/configuration#overriding-third-party-metadata